docs/src/main/asciidoc/security-oidc-bearer-authentication-concept.adoc

```diff
@@ -6,7 +6,9 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 [id="security-oidc-bearer-authentication-concept"]
 = OIDC Bearer authentication
 include::_attributes.adoc[]
-:categories: security
+:categories: security,web
+
```

Comment:
@rolfedh - Only saw this PR post-merge... The linebreak on line 10 breaks the preamble on the index page. Fixed in [PR 30665](https://github.com/quarkusio/quarkus/pull/30665).